src/components/Icon.tsx

```diff
@@ -813,6 +813,21 @@ export const Icons = {
       fillRule="evenodd"
     />
   ),
+  bxTime: (
```

Comment:
question: where does this `bx` prefix come from? Don't see that anywhere else 🤔

Reply:
I pulled it from the name in the figma file, wasn't sure if it was a new convention there. I can remove it, it's been a while since I added Beam icons